workers/repomirrorworker/test/test_repomirrorworker.py

```diff
@@ -2,6 +2,7 @@
 import mock
 import json
 from functools import wraps
+from mock import patch
```

Comment:
can you import this `from unittest.mock import patch` instead?
`mock` is a dependency that we no longer need now that python's standard unittest package provides the same capabilities

Reply:
updated